### ISSUE TITLE
Cannot connect to websocket server

### ISSUE THREAD

**Issue by huangzhtao** [KlausSchaefers/quant-ux-websocket#1]
I run quant-ux on my own server, there is no problem with the back and front end, but I can't connect to this WebSocket server.

There are two errors in the browser console:

1. WebSocket connection to 'ws://XXX.XXX.XXX.XXX:8086/?app=62a9da95e962c64808c19b66&jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyb2xlIjoidXNlciIsImlzcyI6Ik1BVEMiLCJpZCI6IjYyYTlkYTdkZTk2MmM2NDgwOGMxOWI2NSIsImV4cCI6MTY1NTkwNjYzMSwiZW1haWwiOiJodWFuZ3podGFvQDE2My5jb20ifQ.t6x49hhDtFNJSjmUwobUoABtwi_R_FN89KuAEUXnpiI' failed: 
value @ lang.js:15
value @ design.ad5e8c98.js:16
buildCanvas @ design.ad5e8c98.js:16
(anonymous) @ design.ad5e8c98.js:16

2. WebSocketService.init() > cannot init Event {isTrusted: true, type: 'error', target: WebSocket, currentTarget: WebSocket, eventPhase: 2, …} CoreUtil.js:21

And there are two errors in WebSocket server logs:
1. authenticate() > User not allowed to access 62a9da95e962c64808c19b66
2. Cannot autheicate request

Any solution to my problems?

Thanks for your great job!

**Comment by KlausSchaefers** [comment]
Hi,

you need to configure the web socket server to check authentication against the correct Quant-UX backend. This can happen in the conference.json

```
{
  "QUX_SERVER": "http://yourback:8081/",
  "PORT": 8086,
  "LOG_LEVEL": 2
}
```

or via a ENV variable

```
QUX_SERVER
```

Does this help?

**Comment by huangzhtao** [comment]
Is it a mistake for the configure file name as you mentioned (config.json not conference.json)?

I forgot to mention that I used my Dockerfile to build the WebSocket server before because when running your created docker image from docker hub has an error and cannot start the container.
To run the created docker image from docker hub, the command is: `docker run -p 8086:8086 --name quant-ux-ws -e QUX_SERVER=http://XXX.XXX.XXX.XXX:9080/ klausenschaefersinho/quant-ux-websocket:latest`.  
It has the following error:
```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/usr/src/quant-ux-ws/quant-ux-ws/build/build/src/qux-websocket-server.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

After your reply, I succeed to run the WebSocket server without docker and the correct backend server configuration. Thank you.

And I also succeed to test building a docker image with a pre-configured config.json, but still failed with the QUX_SERVER environment variable. I will continue to test this later.
